projects/templates/src/lib/components/po-page-background/po-page-background.component.spec.ts

```diff
@@ -104,9 +108,15 @@ describe('PoPageBackgroundComponent:', () => {
     it('onChangeLanguage: should emit `selectedLanguage`', () => {
       spyOn(component.selectedLanguage, 'emit');
       component.onChangeLanguage();
-
       expect(component.selectedLanguage.emit).toHaveBeenCalled();
     });
+
+    it('setLanguageOptions: should emit `languageList`', () => {
```

Comment:
Pela descrição do It, esta indicando que "deve emitir languageList" porém esta atribuindo novo valor para selectLanguageOptions a partir do languageList

Sugestão:
```
it('setLanguageOptions: should set `selectLanguageOptions` with `languageList`', () => {
      const languages: Array<PoLanguage> = [{ description: 'portugues', language: 'pt' }];
      component.languagesList = languages;
      component.setLanguageOptions();
      expect(component['_selectLanguageOptions']).toEqual([{ label: 'portugues', value: 'pt' }]);
});
```

Reply:
sugestão aceita